File: educational-core/src/com/jetbrains/edu/learning/stepik/courseFormat/StepikSectionRemoteInfo.java
package com.jetbrains.edu.learning.stepik.courseFormat;

import com.google.gson.annotations.Expose;
import com.jetbrains.edu.learning.courseFormat.remote.RemoteInfo;

public class StepikSectionRemoteInfo implements RemoteInfo {
  @Expose
  private int myId;

  public int getId() {
    return myId;
  }

  public void setId(int id) {
    myId = id;
  }
}

File: educational-core/src/com/jetbrains/edu/learning/stepik/courseFormat/StepikSectionRemoteInfo.kt
package com.jetbrains.edu.learning.stepik.courseFormat

import com.jetbrains.edu.learning.courseFormat.remote.RemoteInfo
import java.util.*

class StepikSectionRemoteInfo : RemoteInfo {
  var id: Int = 0
  var units: MutableList<Int> = mutableListOf()
  var courseId: Int = 0
  var position: Int = 0
  var updateDate = Date(0)
}
